apps/extension/client/src/initiateEditor.ts

```diff
@@ -284,7 +296,23 @@ const machine = createMachine(
                 });
               },
             ),
-          ),
+          );
+
+          const extensionErrorDisposable = registerDisposable(
+            extensionContext,
+            languageClient.onNotification('extractionError', ({ message }) => {
+              sendBack({
+                type: 'EXTENSION_ERROR',
+                message,
+              });
+            }),
+          );
```

Comment:
After renaming the notification this disposable should match the notification's name - so it should be `extractionErrorDisposable`. I would probably also rename the `EXTENSION_ERROR` to `EXTRACTION_ERROR`